docs: Suggest gh manual triggers (#1605)
semantic-release/semantic-release: docs/recipes/github-actions.md

## Trigger semantic-release on demand

### Using GUI:
You can use [Manual Triggers](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) for GitHub Actions.

### Using HTTP:

> Use [`repository_dispatch`](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#repository_dispatch) event to have control on when to generate a release by making an HTTP request, e.g.:

Author: I updated this link also, please confirm

Reviewer: Looks good 👍🏼

$ curl -v -H "Accept: application/vnd.github.everest-preview+json" -H "Authorization: token ${GITHUB_TOKEN}" https://api.github.com/repos/[org-name-or-username]/[repository]/dispatches -d '{ "event_type": "semantic-release" }'
```

> If you'd like to use a GitHub app to manage this instead of creating a personal access token, check github's [Manual Triggers](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/)

Reviewer: Please do not remove the existing apps. Actions are not yet available for GitHub Enterprise Server users.

Good idea to add the new workflow dispatch trigger though, I would add that paragraph above the existing one, and maybe rephrase the existing one to be recommended for GHES users.

Also maybe rephrase the new paragraph to something like this? 

```suggestion
You can also use [Manual Triggers](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) for GitHub Actions.
```
